datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1057,6 +1069,20 @@ impl LogicalPlan {
                     }) => {
                         write!(f, "DropTable: {name:?} if not exist:={if_exists}")
                     }
+                    LogicalPlan::TransactionStart(TransactionStartNode {
```

Comment:
The other variants seem to use the name name for the variant and the structure they contain. I wonder if we should call this `TransactionStart` rather than TransactionStartNode` 🤔 

```suggestion
                    LogicalPlan::TransactionStart(TransactionStart {
```